openapi/endpoints/simulator/models/fxAmount.yaml

```diff
@@ -6,7 +6,7 @@ required:
 properties:
   amount:
     type: string
-    description: Transaction amount in a different currency. This can be generated by converting the `transactionAmount`
+    description: Transaction amount as an integer in the smallest denomination for the given currency. This can be generated by converting the `transactionAmount`
```

Comment:
```suggestion
    description: Transaction amount as an integer in the smallest denomination for the given currency. This can be generated by converting the `transactionAmount`
    example: 134
```

Reply:
Doesn't work. Only pattern is supported and that adds too much verbage.

Reply:
🙃 gotcha